src/coreComponents/physicsSolvers/fluidFlow/IsothermalCompositionalMultiphaseFVMKernels.hpp

```diff
@@ -908,6 +986,12 @@ class FaceBasedAssemblyKernel : public FaceBasedAssemblyKernelBase
   typename STENCILWRAPPER::IndexContainerViewConstType const m_seri;
   typename STENCILWRAPPER::IndexContainerViewConstType const m_sesri;
   typename STENCILWRAPPER::IndexContainerViewConstType const m_sei;
+
+  /// Flag to specify whether use C1-PPU or not
```

Comment:
This could belong to [``FluxApproximationBase.hpp``](https://github.com/GEOS-DEV/GEOS/blob/develop/src/coreComponents/finiteVolume/FluxApproximationBase.hpp)

Reply:
i moved not this one but the ones from the solver, this is what you meant, right?